.changeset/fuzzy-mugs-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": patch
+---
+
+Remove vertical and horizontal scroll bars from block preview iframe in cms-admin
```

Comment:
This changeset is explicitly for the `cms-admin` package, so there is no need to mention that 🙂 

```suggestion
Remove vertical and horizontal scroll bars from block preview iframe
```

Reply:
Updated